src/tests/ftest/control/daos_server_helper.py

```diff
@@ -34,17 +36,23 @@ def test_daos_server_helper_format(self):
         :avocado: tags=DaosPrivHelperTest,test_daos_server_helper_format
         """
         # Verify that daos_server_helper has the correct permissions
-        self.log.info("Checking daos_server_helper binary permissions")
-        file_stats = os.stat("/usr/bin/daos_server_helper")
+        # Get the result remotely with os.stat so the format is compatible with local code
+        self.log_step("Verify daos_server_helper binary permissions")
+        helper_path = os.path.join(self.prefix, "bin", "daos_server_helper")
+        cmd = f"python3 -c 'import os; print(os.stat(\"{helper_path}\").st_mode)'"
+        result = run_remote(self.log, NodeSet(self.hostlist_servers[0]), cmd)
+        if not result.passed:
+            self.fail("Failed to get daos_server_helper mode")
+        mode = int(result.joined_stdout)
```

Comment:
From https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-15503/3/artifact/Functional%20Hardware%20Medium/control/daos_server_helper.py/job.log
```
2024-11-14 23:10:18,829 test             L0471 INFO | ==> Step 2: Verify daos_server_helper binary permissions [elapsed since last step: 0.00s]
2024-11-14 23:10:18,829 run_utils        L0470 DEBUG| Running on wolf-142 with a 120 second timeout: python3 -c 'import os; print(os.stat("/usr/bin/daos_server_helper").st_mode)'
2024-11-14 23:10:19,029 run_utils        L0334 DEBUG|   wolf-142 (rc=0): 35304
```
So we get a string similar to if we ran `os.stat` locally